docker-compose.yaml

```diff
@@ -0,0 +1,29 @@
+version: "2.2"
+services:
+  backend:
+    container_name: express-server
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    working_dir: /usr/src/app
```

Comment:
Is this the root of your app? Why is it under /usr/src/app?

Reply:
Yes, it is the root inside the container. I followed the same approach used in the [docker-compose.yaml](https://github.com/teamdigitale/io-backend/blob/master/docker-compose.yml) and the [dockerfile](https://github.com/teamdigitale/io-backend/blob/master/Dockerfile) files of the [app IO backend](https://github.com/teamdigitale/io-backend).